src/js/components/DataClearFilters/stories/Simple.js

```diff
@@ -0,0 +1,66 @@
+import React, { useContext } from 'react';
+
+import {
+  Box,
+  Data,
+  DataClearFilters,
+  DataContext,
+  DataFilter,
+  DataFilters,
+  DataSearch,
+  DataSummary,
+  DataTable,
+  SelectMultiple,
+  Toolbar,
+} from 'grommet';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Simple = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box pad="medium">
+    <Data
+      data={DATA}
+      total={DATA.length}
+      properties={{
+        location: { label: 'Location' },
+      }}
+    >
+      <DataToolbar />
+      <DataSummary />
+      <DataTable columns={columns} />
+    </Data>
+  </Box>
+  // </Grommet>
+);
+
+const DataToolbar = () => {
+  const { filteredTotal, total } = useContext(DataContext);
+
+  return (
+    <Toolbar align="end">
+      <DataSearch placeholder="Search" />
+      <DataFilters updateOn="change">
+        <DataFilter
+          property="location"
+          // override theme margin to align with search + filter
+          contentProps={{ margin: { bottom: 'none', top: 'xsmall' } }}
+          margin="none"
+        >
+          <SelectMultiple
+            placeholder="Select location"
+            options={['Boise', 'Fort Collins', 'Palo Alto', 'San Francisco']}
+            name="location"
+          />
+        </DataFilter>
+      </DataFilters>
+      {filteredTotal !== total ? <DataClearFilters kind="toolbar" /> : null}
```

Comment:
@halocline @MikeKingdom @jcfilben I have applied `kind="toolbar"` here but that's not the default treatment for the one in DataFilters. What treatment do we want? If `toolbar`, how should we adjust the one built-in to DataFilters?

Reply:
I think we want the toolbar styling for both from what I recall

Reply:
Agree to wanting it on both. Grommet doesn't currently have a `toolbar` button kind, that's just an HPE theme thing. Some questions I have then:
- Should we introduce a `toolbar` kind into Grommet theme so we can automatically apply it to the built-in DataFilters? 
- If so, how do we do that since the base.js theme doesn't currently operate on the newer `kind` logic?
- If we don't actually add a `toolbar` kind to base theme, is it still appropriate to add the prop internally to Grommet so the HPE theme can pick up on it?

Reply:
@taysea can you schedule call to discuss? I'd like to brainstorm some options. One of which might be a theme property accepting any button props and getting spread onto the button `...buttonProps`

Reply:
Great, I scheduled a call for tomorrow.

Reply:
After discussion, we will have a theme object that accepts `kind` which can be any button kind.

Reply:
Added this in this commit: https://github.com/grommet/grommet/pull/7025/commits/ca740f5fc3bb691eade57ea0337f159f38d430b1